style-guide.md

```diff
@@ -0,0 +1,30 @@
+# Smart Contract Vulnerability Style Guide
+This style guide outlines the formatting and content expectations for contributions to this repository, focusing on vulnerabilities in smart contracts deployed on Ethereum Virtual Machine [(EVM)-compatible](https://ethereum.org/en/developers/docs/evm/) chains.
+
+## Document Structure
+- *Markdown (.md) Files:* Content will primarily be authored in [markdown format](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) for improved readability and version control.
+- *Consistent Naming:* Use descriptive file names that convey the vulnerability discussed. Examples: `unsupported-opcodes.md`, `default-visibility.md`.
+- *Heading Hierarchy:* Utilize clear heading levels (##, ###, etc.) to structure content and improve navigation. 
+
+## Content Guidelines
+- *Vulnerability Type:* Identify the type of vulnerability at the beginning of the document (eg. Unsupported Opcodes, Reentrancy, Access Control).
+- *Technical Explanation:* Provide a concise technical explanation of the vulnerability, including potential impact and exploit scenarios. Use code snippets where necessary to illustrate the issue.
+- *Affected Chains:* Specify which EVM-compatible chains are susceptible to the vulnerability. Highlight any chain-specific considerations. 
```

Comment:
I would mark this as optional since it's only necessary if it doesn't affect all EVM chains (which most probably do)

Reply:
yes